### ISSUE TITLE
README creation for included scripts

### ISSUE THREAD

**Issue by HughP** [rtxanson/homebrew-language-technology#3]
What about scripts for [PRAAT](http://www.fon.hum.uva.nl/praat/) or [ELAN](https://tla.mpi.nl/tools/tla-tools/elan/) is there a recommendation for how to run a README so that future users know which publications have used particular scripts? Ideally there would be a pull request form person who publish using these scripts telling the README to also include their publication in the citation set of items.

**Comment by rtxanson** [comment]
Are PRAAT and ELAN things that need a system-level package manager to install?

**Comment by HughP** [comment]
No, they are not, but it does not seem that they shouldn't. That is, PRAAT and ELAN could be install with a system installer and then all the secondary scripts which are used with them could be given a systematic location.
